clio-integration-test/src/test/resources/docker-compose/docker-compose-jenkins.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  clio-jenkins-integration-test:
+    image: broadinstitute/clio-integration-test:${CLIO_DOCKER_TAG}
+    environment:
+      ELASTICSEARCH_HTTP_HOST: elasticsearch1.gotc-dev.broadinstitute.org
+    network_mode: "host"
+  clio-server:
+    image: broadinstitute/clio-server:${CLIO_DOCKER_TAG}
+    environment:
+      clio.server.elasticsearch.http-hosts.0.hostname: elasticsearch1.gotc-dev.broadinstitute.org
```

Comment:
I'm surprised this works with the dots, we had to change it to [pass through JAVA_OPTS](https://github.com/broadinstitute/clio/blob/develop/clio-integration-test/src/test/resources/docker-compose/docker-compose.yml#L12) in the other docker-compose file to get around the dots-in-name problem. Maybe a difference in how shell expansion works between our local Docker versions and whatever runs on Jenkins.

Can you change it to use the JAVA_OPTS style, so we're consistent across both setups and guarded against the shell behavior changing out from under us?